vars/external_vars.yml.sample

```diff
@@ -13,9 +13,12 @@ INFLUXDB_PASSWORD: 'project'
 
 # Password for the grafana user who will connect to the grafana database
 INFLUXDB_GRAFANA_PASSWORD: 'grafana'
-#
+
 # Version of grafana you want to install
 GRAFANA_SOURCE_FILENAME: 'grafana-1.8.1'
 
+# Port for grafana nginx virtual server
+GRAFANA_NGINX_PORT: 8080
```

Comment:
Merged and fixed manually: https://github.com/bitmazk/ansible-digitalocean-influxdb-grafana/commit/9b53cfb1f90ba78f8657104f32a2c219b1679d4a